mm2src/coins/lightning/ln_utils.rs

```diff
@@ -69,14 +69,12 @@ pub async fn init_persister(
 }
 
 pub async fn init_db(ctx: &MmArc, ticker: String) -> EnableLightningResult<SqliteLightningDB> {
-    let db = SqliteLightningDB::new(
-        ticker,
-        ctx.sqlite_connection
-            .ok_or(MmError::new(EnableLightningError::DbError(
-                "sqlite_connection is not initialized".into(),
-            )))?
-            .clone(),
-    );
+    // TODO db_id
+    let shared = ctx.sqlite_conn_opt(None).or_mm_err(|| {
```

Comment:
Can we use `lp_coins::account_db_id` here?

Reply:
yep, done